src/main/java/com/snowflake/kafka/connector/internal/telemetry/SnowflakeTelemetryServiceV1.java

```diff
@@ -109,6 +139,25 @@ public void reportKafkaPartitionStart(final SnowflakeTelemetryBasicInfo pipeCrea
     send(TelemetryType.KAFKA_PIPE_START, msg);
   }
 
+  /**
+   * JsonNode data is wrapped into another ObjectNode which looks like this:
```

Comment:
What's the advantage of having nested json instead of flat schema?

Reply:
I don't see any advantage per se apart from having a distinct hierarchy but I wanted to keep the same structure since otherwise we would have to run two different queries if we change and put it into outer node.